src/boot/.babelrc

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Is this what you were referencing in your PR description about the "client's boot script has a different..."? I just want to be sure I understand. I'm feeling more like we need to have a checklist for updating this later as now the browser targets are duplicated here and I could see missing this config entirely (i.e. only seeing the root `.babelrc` later).

Reply:
> Is this what you were referencing in your PR description about the "client's boot script has a different..."?

Yes, exactly. We might not ever need to update this - since the amount of code in the boot script is small it doesn't matter so much if we were to compile it down to ES5 for years to come.